packages/components/Tabs/src/TabsItem.types.ts

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { ViewProps } from 'react-native';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import type { IViewProps } from '@fluentui-react-native/adapters';
+import { IFocusable, IPressableState } from '@fluentui-react-native/interactive-hooks';
+import { ITextProps } from '@fluentui-react-native/text';
+import { IconProps } from '@fluentui-react-native/icon';
+import { IButtonProps, IButtonTokens } from '@fluentui-react-native/button';
+
+export const tabsItemName = 'TabsItem';
+
+export interface TabsItemInfo extends IPressableState {
+  /**
+   * TabsItem icon.
+   */
+  icon?: boolean;
+
+  /**
+   * TabsItem text.
+   */
+  headerText?: boolean;
+
+  /**
+   * Indicates if TabsItem is selected.
+   */
+  selected: boolean;
+
+  /**
+   * Key for tabItem
+   */
+  key: string;
+}
+
+export interface TabsItemState {
+  info: TabsItemInfo;
+}
+
+export interface TabsItemProps extends IButtonProps {
```

Comment:
Should we use IButtonProps here? Or instead list the props for the individual slots (like icon, text, etc.)?

Reply:
Just updated so we're not using IButtonProps anymore